app/src/main/res/values/strings.xml

```diff
@@ -928,4 +928,14 @@
   <!-- App update notification -->
   <string name="app_update_notification_subtext">Please update your Simple app in the Google Play Store as soon as possible.</string>
   <string name="critical_app_update_notification_subtext">Important updates available. Update your Simple app from the Google Playstore.</string>
+  
+  <!-- Critical app update dialog -->
+  <string name="critical_update_required_dialog_title">Critical update required</string>
+  <string name="critical_update_required_dialog_content">Update now for critical security changes and latest features</string>
+  <string name="critical_update_required_dialog_error_waning">App security is compromised</string>
+  <string name="critical_update_required_dialog_update_button_text">UPDATE NOW</string>
+  <string name="critical_update_required_dialog_help_section_title">Need help?</string>
+  <string name="critical_update_required_dialog_help_section_content">Tap on the button below to get help from our friendly support team</string>
+  <string name="critical_update_required_dialog_help_section_phone_number">+91 8397847298</string>
```

Comment:
Do we have same number throughout all countries? Has an Indian code to it as well. Why did we decide to hardcode this?

Reply:
This is just for the layout file. I will change it to `tools`

Reply:
We will change this dynamically. I have just set this here for the layout file.

Reply:
Instead of having a random number, can we keep the field empty in the layout file and update programmatically when we receive something based on country then?

Reply:
+1 on that, we should not be using a random number, since we are not sure if this number is operational or not. Either way, if the number is not loaded we simply hide the bottom section.  So we don't have to provide a text value here.